### PR TITLE
Update install instructions with preliminaries for MacOS and Ubuntu

### PR DESCRIPTION
**Pull request by kokipsiho**

### CODE REVIEW

**Comment by naconner** [comment]
Thanks. :+1: 

By the way, there's also: https://github.com/lamassu/lamassu-machine/wiki/Running-v3-locally-on-Debian-or-Ubuntu

**Comment by joshmh** [comment]
@naconner Yep, but I think we should keep it all in Install.md. It gets versioned with the software and there's just one place to look.

File: INSTALL.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+## Packages
+
+```
+sudo apt-get update
+sudo apt-get install build-essential cmake libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev libpcsclite-dev
+```
+
+## OpenCV
+
+Download the OpenCV source archive and unpack it. In source folder run following:
```
Comment (joshmh) [inline INSTALL.md]: Let's add a link to the source archive or at least to the releases page.

Comment (kokipsiho) [reply INSTALL.md]: Added link to releases page.

**Pull request merged**